stats/src/main/java/com/facebook/stats/mx/JVMStatsExporter.java

```diff
@@ -49,6 +54,34 @@
     .add(SimpleType.DOUBLE)
     .build();
 
+  private static final Set<String> NUMERIC_STRING_TYPES = ImmutableSet.<String>builder()
```

Comment:
What's all this about? The diff is for passing `StatsNameBuilder`, but this seems to be adding some new functionality? Can the `NUMERIC_STRING_TYPES` change be in a separate diff, if we need it?